.eslintrc.js

```diff
@@ -27,7 +27,6 @@ module.exports = {
         'prefer-arrow-callback': 'off',
 
         // The following rules cause problems for our existing tests, so they are disabled for now:
-        'mocha/no-mocha-arrows': 'off',
```

Comment:
Removed this because this PR removes arrow functions in favor of `function ()`

Reply:
perfect!